### ISSUE TITLE
Production ready?

### ISSUE THREAD

**Issue by pierreozoux** [contiv/volplugin#377]
I'm planning to use this in production in the coming months.
I know it is currently alpha.

Can anybody make a list of possible issues that I can face doing so? I'm ready to spend time trying and debugging.

**Comment by erikh** [comment]
I'll try to get back to you later today with this information. I'd say it's beta quality at this point.

**Comment by pierreozoux** [comment]
Beta is already great :)

Maybe we can put a list of other issues, and once they are done, we can close this, let me know.

**Comment by erikh** [comment]
Yep I have some milestones (check the tab under the issues list) but I should probably formalize the plan a little bit. Thanks for the kick in the pants. :)

**Comment by erikh** [comment]
I'm still finalizing the schedule here, but Pierre, I'd love to make sure we can give you what you want. Please feel free to email me (on my github page) and we can discuss your priorities as well as ours and find the right intersection.

**Comment by raarts** [comment]
Did this happen? I am investigating using this in production too.

**Comment by erikh** [comment]
No, in my opinion, this product is not ready for production.
